_pytest/nodes.py

```diff
@@ -178,15 +179,23 @@ def add_marker(self, marker):
         elif not isinstance(marker, MarkDecorator):
             raise ValueError("is not a string or pytest.mark.* Marker")
         self.keywords[marker.name] = marker
+        self._markers.update([marker])
+
+    def find_markers(self, name):
+        """find all marks with the given name on the node and its parents
+
+        :param str name: name of the marker
+        """
```

Comment:
for the reference docs I'd like to mention that this is a generator yielding the marks instead of the very vague "find all marks"

Reply:
fixed, thanks for the note